artifact/compressor_zstd.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
```

Comment:
We need a way to compile this out, since some people want to save on binary size and dependencies. Can you add some build options similar to [this](https://github.com/mendersoftware/mender-artifact/blob/master/artifact/compressor_lzma.go#L14-L15)?

Reply:
Done